### ISSUE TITLE
data loss editing in 2 web instances

### ISSUE THREAD

**Issue by andaag** [Automattic/simplenote-electron#1075]
#### Steps to reproduce
1. I was editing a note at work. And afterwards closed my laptop and brought it home. (With more than enough time for it to in theory be synced, unless sync was down).
2. I opened the same file on my home computer, and see it hasn't been synced. I needed to add information there so I put a url on the top of the note.
3. I opened my laptop in an attempt to get the note to sync, and see instead the changes I made on my home computer. Not anything I've done on the laptop. When checking version history, I see only 2 versions on the laptop.
4. Going back to my home computer I see tons of changes in the change history, including the one with the changes I wanted to make. That was ignored.

Change history on main computer:
VERY man. Including multiple during the edit that I seem to have lost. Between 3:46pm and 3:48 pm I have 10 change history notes. And they detail the edits I was missing. The earliest being november when I created the note.

On my laptop:
Change history has 3 entries, all from the last 24h:
Initial 10:40am
Second 3:48pm
Latest (the edit I made on the home computer, in sync)

#### What I expected
I was expecting the change I made to be added to the note without overwriting the other changes.

#### What happened instead
The url I added overwrote the changes I made (that for some reason hadn't been synced properly). Meaning I lost data since sync was down.

#### Simplenote version
1.3.2 on both computers, running snap package

#### OS version
Ubuntu 18.10

#### Screenshot / Video
N/A

Example of note before editing started:
```
## TODO:

- Stream joining - see if we can do it locally with ktor embedded and querying other hosts. Doable in transform method?!...
```

Change I made at work:
Example of note before editing started:
```
## TODO:

### Stream joining - see if we can do it locally with ktor embedded and querying other hosts. Doable in transform method?!...
```

Url I added at home computer:

Example of note before editing started:
```
## TODO:

http://link.I.Found

- Stream joining - see if we can do it locally with ktor embedded and querying other hosts. Doable in transform method?!...
```
At this point the latest on both is the revision above. Edit #2 of these have disappeared.

**Comment by roundhill** [comment]
Sounds like this could related to this issue in `node-simperium`: https://github.com/Simperium/node-simperium/issues/68

**Comment by andaag** [comment]
On https://app.simplenote.com/ it looks like the same as my home computer.

**Comment by Majubs** [comment]
I've had a similar issue with this steps:

1. I edited a note on my Linux Mint 19 (Simplenote 1.4.1) notebook, while offline, with no internet connection
2. Two days later, with internet connection, I opened the notebook for it to sync
3. Tried to open the note on my Android device (Simplenote 1.7.0), but had only the version of the note from before editing offline
4. Tried to open on a Windows computer (Simplenote 1.5.0) I had just installed the app, so fresh login, but still only the old version of the note
5. Edited the note on the notebook to see if it would trigger sync, but no success
6. Logged out of the notebook, and when logging in again I had lost the edits I made while offline

After this, the note is syncing properly from any device I edit it.
